skimage/measure/_regionprops.py

```diff
@@ -657,10 +657,10 @@ def regionprops(label_image, intensity_image=None, cache=True):
         bounding box.
     **image** : (H, J) ndarray
         Sliced binary region image which has the same size as bounding box.
-    **inertia_tensor** : (2, 2) ndarray
+    **inertia_tensor** : (Ndim, Ndim) ndarray
         Inertia tensor of the region for the rotation around its mass.
     **inertia_tensor_eigvals** : tuple
-        The two eigen values of the inertia tensor in decreasing order.
+        The N eigen values of the inertia tensor in decreasing order.
```

Comment:
The `N` here is not necessary. It could be:
``` The eigenvalues of the inertia tensor in decreasing order.```

Reply:
Done!